numberformat/diff.emu

```diff
@@ -1091,6 +1091,11 @@
         1. Let _literal_ be ParseText(_text_, |StringNumericLiteral|).
         1. If _literal_ is a List of errors, let _mv_ be ~not-a-number~.
         1. Else, let _mv_ be the StringIntlMV of _literal_.
+        1. If _mv_ ≥ 1<sup>1000</sup>, then let _mv_ be ~positive-infinity~.
+        1. If _mv_ ≤ -1<sup>1000</sup>, then let _mv_  be ~negative-infinity~.
+        1. If _mv_ > -1<sup>-1000</sup> < 0, then let _mv_ be ~negative-zero~.
```

Comment:
What is the condition " _mv_ > -1<sup>-1000</sup> < 0"? This does not look like a valid logical expression.